content/en/docs/concepts/services-networking/service.md

```diff
@@ -241,9 +241,18 @@ There are a few reasons for using proxying for Services:
   on the DNS records could impose a high load on DNS that then becomes
   difficult to manage.
 
+Below we'll describe how various kube-proxy implementations work.  In general
```

Comment:
nit ([avoid using “we”](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we)):
```suggestion
Later in this page you can read about various kube-proxy implementations work. Overall,
```